plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/e4/DBeaverStackRenderer.java

```diff
@@ -271,6 +273,21 @@ public void widgetSelected(SelectionEvent e) {
                         DBWorkbench.getPlatformUI().copyTextToClipboard(DBUtils.getObjectFullName(object, DBPEvaluationContext.UI), false);
                     }
                 });
+
+                final MenuItem addBookmarkItem = new MenuItem(menu, SWT.NONE);
```

Comment:
1. This item has to be only for EntityEditor.
2. Do not add yet another "Add Bookmark" string. Use command `command.org.jkiss.dbeaver.core.navigator.bookmark.add` label. See ActionUtils.